<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTLS_Widget</class>
 <widget class="QWidget" name="RTLS_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTLS_Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget1">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Tcp">
      <attribute name="title">
       <string>TCP</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="6">
        <widget class="QTextEdit" name="textEdit_read"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="Button_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_Port">
         <property name="text">
          <string>12900</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>服务器 端口号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>服务器 IP：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>157</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_IP">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Serial">
      <attribute name="title">
       <string>Serial Port</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="SerialLabel">
         <property name="text">
          <string>串口选择</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTextBrowser" name="SerialTextBrowser"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="SerialButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="MyComboBox" name="SerialBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Positioning">
      <attribute name="title">
       <string>Positioning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>图形化界面</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableWidget_Tag">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>175</height>
          </size>
         </property>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Tag ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X(m)</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y(m)</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z(m)</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Tag 0</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Tag 1</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Tag 2</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>Tag 3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>Tag 4</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="4" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>Tag 5</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="5" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="5" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>Tag 6</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="6" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="6" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>Tag 7</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="7" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="7" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="7" column="3">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget_Anchor">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>175</height>
          </size>
         </property>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Anchor ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X(m)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y(m)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z(m)</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Anchor 0</string>
          </property>
          <property name="font">
           <font>
            <underline>false</underline>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>-3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>1</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>0.5</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Anchor 1</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>4</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>-1</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>0</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Anchor 2</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>2</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>Anchor 3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>-2</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>3</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string>2.5</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>读取基站坐标</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Ranging">
      <attribute name="title">
       <string>Ranging</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
